internal/sdkprovider/service/kafka/mirrormaker_replication_flow.go

```diff
@@ -122,6 +122,14 @@ var aivenMirrorMakerReplicationFlowSchema = map[string]*schema.Schema{
 		ValidateFunc: validation.IntAtLeast(1),
 		Description:  "Replication factor, `>= 1`.",
 	},
+	"exactly_once_delivery_enabled": {
+		Type:     schema.TypeBool,
+		Optional: true,
+		Default:  false,
+		Description: userconfig.Desc(
+			"Whether to enable exactly-once message delivery. We recommend you set this to `enabled` for new replications.",
```

Comment:
Could we please change the description to this:

`Enable exactly-once message delivery, replicating each message exactly once between clusters to prevent duplicates or data loss. It's recommended to enable this for new replications.`

Reply:
@nosahama @staceysalamon-aiven  this should be changed on the backend

Reply:
Yes sure, we can look into this later and reiterate 👍 
Backend work has already gone in and would need a new round of changes/PR/CI stuff, so I will just log a ticket around this. 
cc @harshini-rangaswamy